practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/model/RoleDTO.java

```diff
@@ -0,0 +1,31 @@
+package uk.ac.ox.ndph.mts.practitioner_service.model;
+
+import java.io.Serializable;
+
+public class RoleDTO implements Serializable {
```

Comment:
This isn't used, and I don't think we'd in the future. Can you remove it?

Reply:
Done